backend/src/services/org/org-service.ts

```diff
@@ -503,10 +509,50 @@ export const orgServiceFactory = ({
     const { permission } = await permissionService.getUserOrgPermission(userId, orgId, actorOrgId);
     ForbiddenError.from(permission).throwUnlessCan(OrgPermissionActions.Delete, OrgPermissionSubjects.Member);
 
-    const membership = await orgDAL.deleteMembershipById(membershipId, orgId);
+    const deletedMembership = await orgDAL.transaction(async (tx) => {
+      const orgMembership = await orgDAL.deleteMembershipById(membershipId, orgId, tx);
 
-    await licenseService.updateSubscriptionOrgMemberCount(orgId);
-    return membership;
+      if (!orgMembership.userId) {
```

Comment:
when `deleteMembershipById` is successful, it returns what was deleted. I'm thinking `!orgMembership.userId` should be `orgMembership.userId` before you update the membership count. Let me know if i misunderstood

Reply:
`userId` might be null. If the userId is null, then we can't run the rest of the function and remove project memberships & keys. This if statement just checks if a userId is present on the deleted organization membership. If no userId is present, it will update the subscription org member count and return the membership. This is the same logic as we had before.

Let me know if that makes sense!